packages/lesswrong/server/repos/DialogueChecksRepo.ts

```diff
@@ -26,6 +26,24 @@ export default class DialogueChecksRepo extends AbstractRepo<DbDialogueCheck> {
     `, [randomId(), userId, targetUserId, checked, checkedAt])
   }
 
+  async upsertDialogueHideInRecommendations(userId: string, targetUserId: string, hideInRecommendations: boolean) {
```

Comment:
I think we should be using the existing upsert query rather than writing an entirely new one.  (Which, btw, I bet is broken, because the new field is not nullable, but that query doesn't include it at all.)  Sort of the whole point of being able to parameterize the inputs.

Reply:
I looked at these mroe now and decided not to use a common string for them. They're both short enough, and slightly different, such that all that extra string abstraction probably wouldn't do anyone any good

Reply:
I disagree; consider as evidence that the above query is broken (because it's missing the `hideInRecommendations` column & value)